Sources/GoogleCloud/Common/OAuth/OAuthCredentialLoader.swift

```diff
@@ -9,14 +9,25 @@ import Vapor
 
 public class OAuthCredentialLoader {
     public static func getRefreshableToken(credentialFilePath: String, withConfig config: GoogleCloudAPIConfig, andClient client: Client) throws -> OAuthRefreshable {
+        
+        // Tries to access file first
         if let credentials = try? GoogleServiceAccountCredentials(contentsOfFile: credentialFilePath) {
             return OAuthServiceAccount(credentials: credentials, scopes: config.scope, httpClient: client)
         }
-
+        
+        // Then tries to decode as JSON
+        if let credentials = try? GoogleServiceAccountCredentials(json: credentialFilePath) {
+            return OAuthServiceAccount(credentials: credentials, scopes: config.scope, httpClient: client)
+        }
+        
         if let credentials = try? GoogleApplicationDefaultCredentials(contentsOfFile: credentialFilePath) {
             return OAuthApplicationDefault(credentials: credentials, httpClient: client)
         }
-
+        
+        if let credentials = try? GoogleApplicationDefaultCredentials(contentsOfFile: credentialFilePath) {
```

Comment:
@VinceFilary 
This is a duplicate call? Why is this needed? Did you mean to use the json API?

Reply:
Good catch! That is exactly what I meant. Copy and paste curse got the best of me!